LiteCore/Query/N1QL_Parser/n1ql.leg

```diff
@@ -22,8 +22,8 @@ namespace litecore { namespace n1ql {
 %}
 
 
-n1ql = _
-    s:selectStatement _ !.            { $$ = s;}
+n1ql = y_
```

Comment:
Why did the grammar syntax change? I thought the changes to the tool only affected the generated C code. (I guess I must have missed this when reviewing the `peg` PR, sorry)

I don't want to change the grammar syntax if we can help it, since this is a breaking change for anything using `peg`, not just us.

Reply:
Because `peg` generates function names dependent on the name of the function in the grammar. I believe it's `yy_` + [function name]. So `_` generates a function called `yy__`. I guess if we don't want the grammar of n1ql.leg to change, we could adjust the generation in `peg` to not be `yy_` + [function name].